src/system/theme/index.js

```diff
@@ -90,6 +90,8 @@ export default {
 		text: getColor( 'text', 'secondary' ),
 		heading: getColor( 'text', 'primary' ),
 		background: getColor( 'background', 'primary' ),
+		gold: getVariants( 'color.gold' ),
+		gray: getVariants( 'color.gray' ),
```

Comment:
Isn't gold the same as primary?

Reply:
We want to avoid pointing colors from their names, but more like semantics: Primary, Border, Background, etc.

Reply:
Fixed here: https://github.com/Automattic/vip-design-system/pull/126/commits/6f25718548361d93a84ae08bcafd72a92ae384ff

Reply:
Gold is not the same as primary. This is the gold referenced in Figma: https://github.com/Automattic/vip-design-system/blob/trunk/tokens/valet-core.json#L782